infrastructure/kube/keep-prd/electrumx-statefulset.yaml

```diff
@@ -0,0 +1,76 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: electrumx
+  namespace: default
+  labels:
+    app: bitcoin
+    type: electrumx
+spec:
+  replicas: 2
+  serviceName: electrumx
+  volumeClaimTemplates:
+  - metadata:
+      name: electrumx-data
+    spec:
+      storageClassName: electrumx
+      accessModes: [ReadWriteOnce]
+      resources:
+        requests:
+          storage: 450Gi
```

Comment:
This is obviously the first time I've taken a deep look at our Electrum deployment heh---does it replicate so much data that it needs the same amount of storage as our main node?

Reply:
It doesn't, but I wasn't entirely sure of the ratio and I felt like if something we sideways we may not have enough time.

I figured if it was at least as large as bcoin storage we wouldn't have any issues.  A bit of a sledgehammer.